backend/menzaDatabase/models.py

```diff
@@ -9,6 +9,7 @@ class Diner(models.Model):
     name = models.CharField(max_length=50)
     display_name = models.CharField(max_length=50)
     date = models.DateField(auto_now_add=True)
+    description = models.CharField(max_length=500, default='Placeholder description')
```

Comment:
Change default either to "" or "This diner doesn't have a description" or something similar. Probably "" would be the best so we don't need any display logic.